lib/opensusebasetest.pm

```diff
@@ -5,10 +5,19 @@ use testapi;
 
 # Base class for all openSUSE tests
 
+
+sub clear_console() {
+    # 'ctrl-l' does not get queued up in buffer. If this happens to fast, the
+    # screen would not be cleared
+    script_run "clear";
```

Comment:
Plain `type_string "clear\n"` can be used instead of `script_run` to avoid `wait_idle`, can't it?